dictionary.txt

```diff
@@ -907,3 +907,5 @@ _rp	₎
 \QED	∎	
 \male	♂	
 \female	♀	
+\logicalandbar  ⩞ 	
+\logicalorbar ⩢ 	
```

Comment:
I was able to find these LaTeX shortcuts in reference to these symbols. Perhaps these names could be used instead?

```suggestion
\doublebarvee	⩢	
\doublebarwedge	⩞	
\perspcorrespond	⩞	
```